templates/index.html

```diff
@@ -50,12 +50,34 @@
 <body id="home-page">
   <div id="container">
     <h1 id="app-title">Iqra</h1>
-    <a id="play-link" href="https://play.google.com/store/apps/details?id=com.mmmoussa.iqra&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"><img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge-border.png" /></a>
+    <a id="app-store-link" target="_blank" href="https://itunes.apple.com/us/app/iqra/id1391009396"><img alt="Get it on the Apple Store" src="/static/img/app-store-badge.png"></a>
+    <a id="play-link" target="_blank" href="https://play.google.com/store/apps/details?id=com.mmmoussa.iqra&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"><img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" /></a>
     <a href="#c1"><i class="material-icons more-arrow">keyboard_arrow_down</i></a>
-    <a id="try-webapp-button" href="/app">Try it now!</a>
+    <div id="try-webapp-button-container">
+      <a id="try-webapp-button" href="/app/search">Try it now!</a>
+    </div>
   </div>
   <div id="home-content">
-    <a class="nav-link" name="c1"></a>
+    <nav>
+      <div class="header" name="c1">
+        <a href="/" class="logo-container">
```

Comment:
We're already on the home page here so I think it makes sense to just scroll to the top of the page rather than reloading.